src/shogun/mathematics/linalg/LinalgNamespace.h

```diff
@@ -125,10 +135,7 @@ SGVector<T> from_gpu(const SGVector<T>& vector)
 		return SGVector<T>(data, vector.vlen);
 	}
 	else
-	{
-		SG_SWARNING("Trying to run GPU code without GPU backend registered.\n");
-		return vector;
-	}
+		SG_SERROR("No GPU backend registered.\n");
```

Comment:
@lambday right right my bad. I got this method mixed up with other ones. I'll fix this. Thanks!